libs/crosstarget-utils/src/wasm/time.rs

```diff
@@ -108,3 +109,42 @@ fn now() -> f64 {
         .map(|p| p.now())
         .unwrap_or_else(Date::now)
 }
+
+#[derive(Clone, Debug)]
+pub struct DateTime(js_sys::Date);
+
+impl UtcDateTime for DateTime {
+    fn now() -> Self {
+        Self(js_sys::Date::new_0())
+    }
+
+    fn format(&self, format_str: &str) -> String {
+        let mut chars = format_str.chars();
+        let mut result = String::new();
+
+        while let Some(c) = chars.next() {
+            if c == '%' {
+                if let Some(specifier) = chars.next() {
+                    match specifier {
+                        'Y' => result.push_str(&self.0.get_utc_full_year().to_string()),
+                        'm' => result.push_str(&format!("{:02}", self.0.get_utc_month() + 1)), // JS months are 0-based
+                        'd' => result.push_str(&format!("{:02}", self.0.get_utc_date())),
+                        'H' => result.push_str(&format!("{:02}", self.0.get_utc_hours())),
+                        'M' => result.push_str(&format!("{:02}", self.0.get_utc_minutes())),
+                        'S' => result.push_str(&format!("{:02}", self.0.get_utc_seconds())),
+                        _ => result.push_str(&format!("%{}", specifier)),
+                    }
+                }
+            } else {
+                result.push(c);
+            }
+        }
+
+        result
+    }
```

Comment:
Please don't judge. Any easier alternative that works well with `wasm32-unknown-unknown` is appreciated.

Reply:
`chrono` has `wasmbind` feature:

> `wasmbind`: Interface with the JS Date API for the `wasm32` target.

Reply:
Thanks, was not aware of it. I've fixed it in https://github.com/prisma/prisma-engines/pull/5276/commits/d805d9ab930ab81005aba3b00d7afb5f813c918f.